### PR TITLE
Feature - multi others

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: pages/rops/update/index.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ module.exports = () => {
           data: req.multiStep.values
         }
       };
+      console.log(params.json.data);
```
Comment (lennym) [inline pages/rops/update/index.js:20]:
```suggestion
```

File: pages/rops/update/schema/index.js [hunk 2]
```diff
@@ -367,17 +376,18 @@ module.exports = req => {
...
           value: 'routine-other',
           reveal: {
-            regulatorySubpurposesOther: getOtherField('regulatorySubpurposes', 'routine-other')
+            regulatorySubpurposesOther: getOtherField('regulatorySubpurposes')
           }
         },
         'qc-batch-safety',
         'qc-pyrogenicity',
         'qc-batch-potency',
+        // TODO: specify other
```
Comment (lennym) [inline pages/rops/update/schema/index.js:385]: 👀

Comment (joefitter) [reply pages/rops/update/schema/index.js:385]: yes I added this as a reminder :)

Comment (joefitter) [reply pages/rops/update/schema/index.js:385]: it will need _another_ migration

**Pull request merged**